packages/venia-ui/lib/components/CartPage/PriceAdjustments/ShippingMethods/shippingForm.js

```diff
@@ -82,7 +82,7 @@ const ShippingForm = props => {
                 {!hasMethods ? (
                     <Button
                         classes={{ root_normalPriority: classes.submit }}
-                        disabled={setShippingLoading ? true : null}
+                        disabled={setShippingLoading}
```

Comment:
Took me a second to parse this one because `setShippingLoading` reads like a function, not a boolean.

Up to you if you want to change it or not.

Reply:
Yep, renamed to follow our bool naming convention in cf8fd2e, good call.